security/publish_dispatcher_spec.rb

```diff
@@ -17,8 +17,23 @@
   end
 
   it 'should not be able to access Publish pages as site visitor' do
+    secure_urls = File.readlines('security/secure_public_routes.txt')
```

Comment:
Rename the terminology `secure_public_routes` with `publish_pages_and_scripts` , for consistency with the terminology on https://helpx.adobe.com/experience-manager/dispatcher/using/dispatcher-configuration.html .